pm_icecon/bt/params/amsr2_goddard.py

```diff
@@ -1,4 +1,4 @@
-"""amsr2_goddard.py: bootstrap parameters file.
```

Comment:
I'd like to add more information about when these parameters are most appropriately used!